pyconbalkan/core/static/css/components/event-sidebar.css

```diff
@@ -8,21 +8,37 @@
     top: 0;
     left: 0;
     bottom: 0;
-    z-index: 2;
-    background-color: #fff;
     width: 400px;
-    padding: 30px 30px 30px 70px;
+    padding: 20px 40px 40px 40px;
+    background-image: url(/static/img/sidebar-bg.png);
```

Comment:
Is it possible to use {% url %} from Django here?

Reply:
Probably, I'm a bit rusty, last time I worked with Django templating was 3 years ago ;)

Reply:
Well, it seems it's not possible, the static is only for templates 😄